### PR TITLE
Update page design

### PR DESCRIPTION
**Pull request by Nikolay-Gavazov**
Made the colors and positions on the components. Add date and show more button.

### CODE REVIEW

**Review by Getsov — APPROVED** [review]

**Comment by Nikolay-Gavazov** [comment]
Replaced the icons with the ones from the design.

File: src/app/tabs/tabs.page.html [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
   <ion-tab-bar slot="bottom">
     <ion-tab-button tab="home" href="/home">
-      <ion-icon aria-hidden="true" name="home"></ion-icon>
+      <!-- <ion-icon aria-hidden="true" name="home"></ion-icon> -->
+      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
```
Comment (Getsov) [inline src/app/tabs/tabs.page.html]: Is this svg working when tab is active?

File: src/app/tabs/tabs.page.html [hunk 2]
```diff
@@ -12,17 +15,20 @@
...
     <ion-tab-button tab="notifications" href="/notifications">
-      <ion-icon aria-hidden="true" name="heart-outline"></ion-icon>
+      <ion-icon aria-hidden="true" name="heart"></ion-icon>
       <ion-label>Любими</ion-label>
     </ion-tab-button>
 
     <ion-tab-button tab="profile" href="/profile">
-      <ion-icon aria-hidden="true" name="person"></ion-icon>
+      <!-- <ion-icon aria-hidden="true" name="person"></ion-icon> -->
+      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
```
Comment (Getsov) [inline src/app/tabs/tabs.page.html]: Same here - is this working when highlighting active tab?

File: src/app/tabs/tabs.page.html (hunk 1 above)
Comment (Nikolay-Gavazov) [reply src/app/tabs/tabs.page.html]: Screenshot 2023-11-13 111249

File: src/app/tabs/tabs.page.html (hunk 2 above)
Comment (Nikolay-Gavazov) [reply src/app/tabs/tabs.page.html]: Screenshot 2023-11-13 111321

File: src/app/tabs/tabs.page.html (hunk 1 above)
Comment (Getsov) [reply src/app/tabs/tabs.page.html]: So it’s not working, right? See the design

File: src/app/tabs/tabs.page.html (hunk 2 above)
Comment (Getsov) [reply src/app/tabs/tabs.page.html]: The same here

**Comment by Nikolay-Gavazov** [comment]
Screenshot 2023-11-13 12
Screenshot 2023-11-13 111321

Done

**Comment by Nikolay-Gavazov** [comment]
Screenshot 2023-11-13 111
Screenshot 2023-11-13 111249

Done

**Pull request merged**